custom-stylelint-plugins/css-logical-properties.js

```diff
@@ -0,0 +1,73 @@
+const stylelint = require('stylelint');
+const { report, ruleMessages, validateOptions } = stylelint.utils;
+
+const ruleName = 'custom-rule/no-logical-properties';
+const messages = ruleMessages(ruleName, {
+  expected: logicalProp => `Unexpected usage of logical property "${logicalProp}"`,
+});
+
+const logicalProps = [
+  'inset-block-start',
+  'inset-inline-end',
+  'inset-block-end',
+  'margin-inline',
+  'inset-inline-start',
+  'margin-block-start',
+  'margin-inline-end',
+  'margin-block-end',
+  'margin-inline-start',
+
+  'padding-block-start',
+  'padding-inline-end',
+  'padding-block-end',
+  'padding-inline-start',
+
+  'inline-size',
+  'min-inline-size',
+  'max-inline-size',
+  'block-size',
+  'min-block-size',
+  'max-block-size'
+];
+
+module.exports = stylelint.createPlugin(ruleName, function getPlugin(primaryOption, secondaryOptionObject, context) {
+  return function lint(postcssRoot, postcssResult) {
+    const validOptions = validateOptions(postcssResult, ruleName, {
+      actual: primaryOption,
+      possible: [true],
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    const isAutoFixing = Boolean(context.fix);
+
+    postcssRoot.walkDecls(decl => {
```

Comment:
I'm not entirely clear on what this plugin is correcting for - it seems to be replacing logical properties with physical ones but isn't our best practice to use logical properties instead?

Would you mind adding some comments in here to clarify the goals?

Reply:
I have replaced this with https://github.com/csstools/stylelint-use-logical module which lints for usage of non logical properties.

Reply:
@Rajdeepc once you push the commit containing `stylelint-use-logical`, it looks like we can remove this file?